airflow/contrib/hooks/spark_submit_hook.py

```diff
@@ -492,8 +496,11 @@ def _start_driver_status_tracking(self):
         while self._driver_status not in ["FINISHED", "UNKNOWN",
                                           "KILLED", "FAILED", "ERROR"]:
 
-            # Sleep for 1 second as we do not want to spam the cluster
-            time.sleep(1)
+            # Sleep for n seconds as we do not want to spam the cluster
+            if self._status_poll_interval:
```

Comment:
If we use suggestion from init then we can simply do:
```suggestion
            time.sleep(self._status_poll_interval)
```